ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandServiceIT.scala

```diff
@@ -198,9 +198,9 @@ final class CommandServiceIT extends LedgerTestSuite {
   )(implicit ec => { case Participants(Participant(ledger, party)) =>
     val request = ledger.submitAndWaitRequest(party, Dummy(party).create.command)
     for {
-      _ <- ledger.submitAndWaitForTransaction(request)
+      _ <- ledger.submitAndWaitForTransactionReturningTransaction(request)
```

Comment:
In general, it appears this test case doesn't really need to transform the result, as it's unused.

```suggestion
      _ <- ledger.submitAndWaitForTransaction(request)
```